### ISSUE TITLE
[FEAT] Improve accessibility of docs

### ISSUE THREAD

**Issue by zslade** [moj-analytical-services/splink#1487]
### Is your proposal related to a problem?
There are areas of the Splink docs which could benefit from changes to improve accessibility.

### Describe the solution you'd like
Suggestions from the MOJ Digital Accessibility team, which should be straightforward to implement 🤞:
- Adding descriptive alt text to images
- Making hyperlinked text descriptive in its own right (where possible), e.g. see our [Contributing Guide](), rather than see [here]()
- Ensuring colour ratio of text to background is at least 4.5
- Giving hyperlinks two distinguishing features, e.g. bold *and* underlined

Tools such as [Axe](https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd) and Mac's in-built screen reader VoiceOver can be used to identify other possible improvements.

### Additional context

References for further information:
- https://www.gov.uk/service-manual/helping-people-to-use-your-service/understanding-wcag#meeting-government-accessibility-requirements
- https://www.w3.org/TR/WCAG21/

**Comment by ADBond** [comment]
Related: #642
